CDP4IME/CDP4IME-CE.csproj

```diff
@@ -229,7 +229,7 @@
     <Reference Include="Microsoft.Practices.Prism.Interactivity">
       <HintPath>..\packages\Prism.Interactivity.5.0.0\lib\NET45\Microsoft.Practices.Prism.Interactivity.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Practices.Prism.MefExtensions">
+    <Reference Include="Microsoft.Practices.Prism.MefExtensions, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I don't know why this has changed.
Can i get some theory on this?

Reply:
I guess when we played with reference specific version between false and true it has changed that line.

Reply:
So, should it stay this way?

Reply:
Yes, I reversed it back, It would not matter anyway, since I do not think will use multiple versions. The package is common for both Addin and IME.